namerd/iface/control-http/src/test/scala/io/buoyant/namerd/iface/HttpControlServiceStreamingTest.scala

```diff
@@ -0,0 +1,73 @@
+package io.buoyant.namerd.iface
+
+import com.twitter.finagle.Name.Bound
+import com.twitter.finagle.http.Request
+import com.twitter.finagle._
+import com.twitter.finagle.naming.NameInterpreter
+import com.twitter.io.{Reader, Buf}
+import com.twitter.util.{Return, Var, Activity}
+import io.buoyant.namerd.NullDtabStore
+import io.buoyant.test.Awaits
+import org.scalatest.FunSuite
+
+class HttpControlServiceStreamingTest extends FunSuite with Awaits {
```

Comment:
i'd prefer that this just be part of the HttpControlServiceTest so that all of the endpoints are enumerated in once place, and then test names should be in the same form...